modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -159,31 +159,31 @@ def wait(self):
         self.drain_call_queue()
         wait(self.future)
 
-    def mask(self, row_indices, col_indices):
+    def mask(self, row_labels, col_labels):
         """
         Lazily create a mask that extracts the indices provided.
 
         Parameters
         ----------
-        row_indices : list-like, slice or label
-            The indices for the rows to extract.
-        col_indices : list-like, slice or label
-            The indices for the columns to extract.
+        row_labels : list-like, slice or label
+            The row labels for the rows to extract.
```

Comment:
in cudf_on_ray/dataframe/dataframe.py the docstring for these functions is something like:

```
row_labels : list of hashable, optional
            The row labels to extract.
```

Should the docstring here be changed to match that one?